documentation/modules/ROOT/pages/configuration/content-based-routing.adoc

```diff
@@ -92,6 +92,10 @@ value.op == 'u' ? 'updates' : null
 |Default
 |Description
 
+|`topic.regex`
```

Comment:
Should we consider adding the configuration property links like we've began doing for the connectors?